src/var/mash.jl

```diff
@@ -0,0 +1,34 @@
+# MinHash
+# ========
+#
+# Functions to MASH distance for MinHash sketches
+#
+# see DOI: 10.1186/s13059-016-0997-x
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+function jaccarddistance(sketch1::MinHashSketch, sketch2::MinHashSketch)
+    d = length(setdiff(sketch1.sketch, sketch2.sketch))
```

Comment:
@kescobo I think the reason it may not have been much faster is the use of `shift!`? I think you can probably do the task you are tying to do in 117 without all the array manipulation, and just keeping track of two indicies.